daisy_workflows/image_build/debian/build_fai.py

```diff
@@ -134,6 +203,11 @@ def main():
 
   if returncode != 0:
     raise subprocess.CalledProcessError(returncode, cmd)
+  if run_sbom_bool:
+    fdisk_output = subprocess.run(['fdisk', '-l', work_dir + '/' + disk_name],
```

Comment:
Why don't you use `sfdisk -J` ? That will give you the output in jason format making your life much simpler.

Reply:
done